File: JSBASIC/145_count_lowercase.js
// 145	145_count_lowercase.js	Count the number of lowercase letters in a given string.

let str = "Hello World JS";
let count = 0;
for (let i = 0; i < str.length; i++) {
    if (str[i] >= 'a' && str[i] <= 'z') count++;
}
console.log("String:", str);
console.log("Lowercase letters:", count);

File: JavaBasic/145_count_lowercase.java
// 145	145_count_lowercase.java	Count the number of lowercase letters in a given string.
public class _145_count_lowercase {
    public static void main(String[] args) {
        String str = "Hello World JS";
        int count = 0;
        for (char c : str.toCharArray()) if (Character.isLowerCase(c)) count++;
        System.out.println("Lowercase count: " + count);
    }
}
